indico/types/workflow.py

```diff
@@ -90,8 +92,8 @@ def __init__(
 
 
 class ComponentFamily(Enum):
-    MODEL = 0,
-    FILTER = 1,
-    TRANSFORMER = 2,
-    REVIEW = 3,
+    MODEL = (0,)
+    FILTER = (1,)
```

Comment:
I don't remember why these are necessary 🤔 but I know they are?

Reply:
yea i noticed that, and figured i best not touch it. black just added parens